pkg/packagekit/autoupdates.jsx

```diff
@@ -359,6 +359,11 @@ export class AutoUpdates extends React.Component {
 
         return (<>
             <div id="autoupdates-settings">
+                {enabled && this.state.backend.parsing_failed &&
+                <Alert isInline
+                       variant="danger"
```

Comment:
danger seems inflated: neither is there anything actually wrong on the system, nor can the user do any dangerous action here. info seems enough?

Reply:
This will alert will emerge from the following scenarios:
*  invalid onCalendar 
* invalid OnUnitInactiveSec
* usage of none of these two

The first two scenarios are actually a problematic configuration, having some typo in the config file or so. 
The last one, is weird, it's a timer unit file without repeating pattern?

@martinpitt therefore i thought it's a danger. I am happy to revert it to info, but am I missing something here?

Reply:
I thought the normal case was just that the custom timer uses a format that we can't parse. Like the `OnCalendar=*-02-* 11:00` that we have in the test suite, which is a perfectly legal `systemd.time(7)` specification, but too complicated for our parser.

Reply:
Indeed. Just noticed.